.travis.yml

```diff
@@ -4,12 +4,17 @@ python:
 - '3.7'
 cache:
   pip: true
+install:
+- sudo apt update
+- DEBIAN_FRONTEND=noninteractive sudo apt install -y latexmk texlive-full dvipng
+- pip install -r requirements.txt
 before_script:
 - openssl aes-256-cbc -K $encrypted_0022eb92da20_key -iv $encrypted_0022eb92da20_iv
  -in .deploy_rsa.enc -out ./.deploy_rsa -d
 - chmod 600 .deploy_rsa && eval $(ssh-agent -s) && ssh-add .deploy_rsa
 script:
-- cd docs && make html
+- cd docs && make latexpdf
+- make html
 after_success:
 - if [[ "$TRAVIS_BRANCH" == "master" ]]; then
     rsync -e 'ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' -ax
```

Comment:
You need to upload the PDF too. I would do something like
```
- scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null build/latex/*.pdf docs@ci.openquake.org:/var/www/docs.openquake.org/crave/
```

it must be done _after_ the rsync